### ISSUE TITLE
Reach out to safe-transmute crew

### ISSUE THREAD

**Issue by rsheeter** [googlefonts/oxidize#11]
We should ping https://github.com/rust-lang/project-safe-transmute, see if/when they might have something ready to use. I believe the answer could influence the choice between zerocopy and view.

Context: discussion on https://github.com/googlefonts/oxidize/issues/10

**Comment by cmyr** [comment]
definitely, and we could discuss possibly contributing to that effort in some way if it would make sense. It might also be worth it to reach out to the zerocopy folks, since I would expect fuchsia to be interested in the safe transmute work as well.
